dark-mode.css

```diff
@@ -177,7 +177,7 @@ body.dark-mode select {
 }
 
 body.dark-mode select option {
-	color: var(--primary-font-color);
+	color: var(--color-white);
```

Comment:
We have a duplicated selector below at line [187](https://github.com/pbaity/rocketchat-dark-mode/pull/53/files#diff-f2e33b3cc2c42b19bde0c0a20119973fR187) that overrides this change.

The style works without the duplication!

![image](https://user-images.githubusercontent.com/19298197/90245065-55465a00-de08-11ea-81ac-27067f1d0698.png)


> Using Linux Desktop

Reply:
Good catch @brunobertoldi 

@UncleSamMericaMan could you remove the duplicate rule on line 187? Not sure how it got in there in the past to begin with. If that works I'll merge